src/components/RoleHeadPanel.js

```diff
@@ -58,7 +58,7 @@ class RoleHeadPanel extends FormPanel {
                             control={<Checkbox 
                                 checked={!!edited && !!edited.isSystem && edited.isSystem}
                                 onChange={event => this.updateAttribute("isSystem", event.target.checked)}
-                                disabled={!!isReadOnly}
+                                disabled={!!edited && !!edited.id}
```

Comment:
why not disabled=True

Reply:
When creating a new Role we want to be able to set `System` to either true or false, correct? As I understand, this checkbox should be disabled only when editing/duplicating a Role.